services/spar/src/Spar/SCIM.hs

```diff
@@ -309,16 +308,15 @@ instance SCIM.UserDB Spar where
               SCIM.serverError "The IdP corresponding to the provisioning token \
                                \was not found"
             Just idpConfig -> pure (idpConfig ^. SAML.idpMetadata . SAML.edIssuer)
-    let uref = SAML.UserRef issuer (SAML.opaqueNameID extId)
+    let uref = traceShowId $ SAML.UserRef issuer (SAML.opaqueNameID extId)
```

Comment:
Oops.  My process always was to remove the `import Debug.Trace`, and then let the compiler lead me to the places to wipe up.  I guess that doesn't work any more unless I tweak `Imports` a little, and then you'll be sad.  I'll try to adapt to `Imports` for a while I guess.

Reply:
A solution I've seen in some other prelude is adding {-# DEPRECATED #-} to traceShow and friends so that it would compile but emit a warning. Would that be better for your workflow?

Reply:
Yes, i like that idea.  I'll make another PR at some point, thanks!